hooks/command

```diff
@@ -172,6 +172,14 @@ fi
 
 # Mount ssh-agent socket and known_hosts
 if [[ "${BUILDKITE_PLUGIN_DOCKER_MOUNT_SSH_AGENT:-$mount_ssh_agent}" =~ ^(true|on|1)$ ]] ; then
+  if [[ -z "${SSH_AUTH_SOCK:-}" ]] ; then
+    echo "+++ 🚨 \$SSH_AUTH_SOCK isn't set, has ssh-agent started?"
+    exit 1
+  fi
+  if [[ ! -f "${SSH_AUTH_SOCK}" ]] ; then
```

Comment:
Can this is `-S` to test for socket? Or perhaps even a different test after the file check that specifically tests the socket-ness of the file?

Reply:
Yeah, good idea @plasticine